src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -9,7 +9,7 @@ private const val POSITION_DISPLAY_SYMBOL = "-"
 fun printPlayResult(result: RacingResult) {
     println("실행 결과")
     result.value
-        .forEach { cars -> printPosition(cars.cars) }
+        .forEach { racingRecord -> printPosition(racingRecord.cars) }
 }
 
 private fun printPosition(cars: List<Car>) {
```

Comment:
코틀린스럽게 작성하면 이렇게 간결하게 작성하는 방법도 있습니다~!
https://kotlinlang.org/docs/scope-functions.html#with

```suggestion
private fun printPosition(cars: List<Car>) = with(StringBuilder()){
    cars.forEach { append("${it.name.value} : ${convertToPositionSymbol(it)}\n") }
    println(this.toString())
}
```

Reply:
오.... 사내에서`with` 사용해보라고 코드리뷰 남겼던 적이 있는데, 이걸 알았더라면 더 깔끔하게 소개해줬을텐데.... 🤣 
지금이라도 알게되서 다행이네요. 😃 
감사합니다.